README.md

```diff
@@ -75,6 +76,16 @@ Or... If you're very fancy:
 
 `export PGPASSWORD=${token} && psql -h localhost -p 5432 -U ${user} -d ${db}`
 
+## SOCKS5 proxy
+
+`dbc` can also open a SOCKS5 proxy to allow any compatible application to connect to all databases through a proxy, thus not requiring to port-forward to a local port.
+
+Simply login to the right environment, and run `dbc proxy`. `dbc` will open a SOCKS5 proxy on `localhost:1080` (port can be configured with `-p`).
+
+Then, setup your database client's proxy settings to target the proxy port, leave `dbc proxy` running as long as you need, and enjoy!
+
+When running Terraform plans locally for databases, you need to use this option. Otherwise, you would have to change your Postgres provider configuration (host and port) each time you need to plan locally. To use the SOCKS5 proxy with the Terraform PostgreSQL provider, set `ALL_PROXY=socks5://localhost:1080` before running your plan.
```

Comment:
I'm assuming there's no way we can manage this `ALL_PROXY` business within the Go code? No worries if not

Reply:
Nope, there is no way of exporting an environment variable from a program to the parent shell.
Moreover, `dbc proxy` has to run in the background in another tab, so either way, you'd need to set this environment variable in another tab, where you'd run your `terraform plan` or whatever